### PR TITLE
docs: update mgt-file component documentation page

### PR DESCRIPTION
**Pull request by musale**
Closes https://github.com/microsoftgraph/microsoft-graph-toolkit/issues/2088

### CODE REVIEW

**Comment by musale** [comment]
CC @sebastienlevert @Mnickii @gavinbarron

File: concepts/toolkit/components/file.md [hunk 1]
```diff
@@ -75,26 +92,33 @@ The `mgt-file` component defines the following CSS custom properties.
...
-  --line2-font-size: 12px;
-  --line2-font-weight: 400;
-  --line2-color: #797775;
-  --line2-text-transform: none;
-
-  --line3-font-size: 12px;
-  --line3-font-weight: 400;
-  --line3-color: #797775;
-  --line3-text-transform: none;
+  /**NOTE: default-* tokens will override anything you set in the component.
```
Comment (sebastienlevert) [inline concepts/toolkit/components/file.md:95]: @musale what are the default tokens available? Only font-family?

Comment (musale) [reply concepts/toolkit/components/file.md:95]: They are not many used in the codebase. We have `default-font-family`, `default-font-weight`.

Comment (sebastienlevert) [reply concepts/toolkit/components/file.md:95]: Please document them in the overall styling article and I'd remove the mention from here.

Comment (musale) [reply concepts/toolkit/components/file.md]: I've removed the mention. I will update them in the overall styling PR.

**Review by gavinbarron — APPROVED** [review]

**Pull request merged**